traversal/scanner/GraphIterator.java

```diff
@@ -45,6 +46,7 @@
 import static com.vaticle.typedb.core.common.exception.ErrorMessage.Internal.RESOURCE_CLOSED;
 import static java.util.stream.Collectors.toMap;
 
+@NotThreadSafe
```

Comment:
Just wanted to make sure this is explicitly noted as not thread safe. Previously, it was only implicit knowledge that it should be used from a single thread.